src/option.rs

```diff
@@ -1,6 +1,4 @@
-//! This module contains the parallel iterator types for options
-//! (`Option<T>`). You will rarely need to interact with it directly
-//! unless you have need to name one of the iterator types.
+//! Parallel iterator types for [options](std::option).
```

Comment:
We'll need to add a markdown link for each of these new links to std we've introduced so they light up in the rendered docs:

```rust
//! Parallel iterator types for [options](std::option).
//! 
//! [std::option]: https://doc.rust-lang.org/std/option/
```